### ISSUE TITLE
Fix missing invoices for customer

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#7235]
Previously, we noticed invoice line items would hit Stripe's limit of 250 (IIRC). We spoke with Stripe and they added us to their beta program so we could increase the limit to 2,000. 

The following [customer](https://app.hubspot.com/live-messages/3888472/inbox/2961543684#email) 🔒 exceeded the limit of 2,000 as they manually inactivated active users. Exceeding this limit broke invoices on Stripe's end so neither we nor they could access the customer's invoice (I asked their customer support). We've since removed the option to manually toggle inactive users. 

We manually created an invoice for the customer that month, but invoices haven't been showing up in their Parabol app since June.

### AC

- [ ] The customer is able to view their invoices in Parabol again

**Comment by nickoferrall** [comment]
After looking into this, I can see that:
- It looks like the company began misbehaving by manually toggling active users in [September 2021](https://dashboard.stripe.com/invoices/in_1JcqhkFLaFINmHnByUWUHCx0). This was the first month that they hit the 250 invoice item limit. 
- When they hit that limit, not all of the payments could be processed. Only payments from 26th August - 23rd Sep were processed and included in the invoice
- The following month, in [Oct 2021](https://dashboard.stripe.com/invoices/in_1Jnj0eFLaFINmHnBiOiGkEn9), only payments from the 12th - 23rd Oct were included
- [23rd July 2022](https://dashboard.stripe.com/invoices/in_1LOh9JFLaFINmHnBCPOgwinf), the limit increased to 2,000. We successfully charged them from 23rd June - 23rd July. The old payments that weren't processed were then queued up, and we began charging them. So, payments go back to the 24th May
- [23rd August](https://dashboard.stripe.com/invoices/in_1LZvstFLaFINmHnBTDPlmSxD): it shows payments from the 23rd August - 23rd July. Then 24th & 23rd May, then 10th May - April 23rd. It goes all the way back to 23rd March
- [23rd September](https://dashboard.stripe.com/invoices/in_1LlAg6FLaFINmHnBkbL8Squf): follows the same pattern and processes payments from 11th Jan to 23rd March

TL;DR: The past few invoices with the 2,000 line limit have processed 2/3 months of old payments that weren't processed because they hit the 250 limit. September 2021 - Jan 11th 2022 haven't been processed yet. 

After two more 2,000 line invoices, we should have processed all the old ones, and the invoices will show up in Parabol again. So, unfortunately, I suggest that we continue to manually create invoices for the next two months. 

@mattkrick what do you think?

**Comment by nickoferrall** [comment]
Hopefully, their invoice in November clears out the queued-up invoices. I'll check their invoice on the 23rd of November to see if it's resolved & close this issue if it is

**Comment by mattkrick** [comment]
Sounds like the bets course of action to me! Crazy how over the top that went...

**Comment by nickoferrall** [comment]
The most recent [invoice](https://dashboard.stripe.com/invoices/in_1M7HjjFLaFINmHnBtk3P2Uib) 🔒 cleared out all of the queued-up invoices from August & Sep 2021. There were only 831 line items in November instead of 2,000, which we were seeing for previous invoices.

[November's invoice](https://action.parabol.co/me/organizations/HdBa4q0t6S) isn't showing up in their billing page. I'll check in December to see if their invoice shows up in the billing page now that the queue has been cleared.
